### PR TITLE
update to appwrite 1.3.0

### PR DESCRIPTION
**Pull request by christyjacob4**
## What does this PR do?

(Provide a description of what this PR does.)

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work.)

## Related PRs and Issues

(If this PR is related to any other PR or resolves any issue or related to any issue link all related PR and issues here.)

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

(Write your answer here.)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
 # Appwrite Node.js SDK
 
 ![License](https://img.shields.io/github/license/appwrite/sdk-for-node.svg?style=flat-square)
-![Version](https://img.shields.io/badge/api%20version-1.2.0-blue.svg?style=flat-square)
+![Version](https://img.shields.io/badge/api%20version-1.2.1-blue.svg?style=flat-square)
```
Comment (Meldiron) [inline README.md:4]: 1.3.0?

File: lib/services/functions.js [hunk 2]
```diff
@@ -51,16 +51,16 @@ class Functions extends Service {
...
+     * @param {string[]} execute
      * @param {string[]} events
      * @param {string} schedule
      * @param {number} timeout
      * @param {boolean} enabled
      * @throws {AppwriteException}
      * @returns {Promise}
      */
-    async create(functionId, name, execute, runtime, events, schedule, timeout, enabled) {
+    async create(functionId, name, runtime, execute, events, schedule, timeout, enabled) {
```
Comment (Meldiron) [inline lib/services/functions.js:63]: Is this nessessary?

File: index.d.ts [hunk 3]
```diff
@@ -2601,7 +2763,7 @@ declare module "node-appwrite" {
      * @throws {AppwriteException}
      * @returns {Promise}
      */
-    getAttribute(databaseId: string, collectionId: string, key: string): Promise<Response>;
+    getAttribute(databaseId: string, collectionId: string, key: string): Promise<string>;
```
Comment (Meldiron) [inline index.d.ts:2766]: Why string?

File: lib/query.js [hunk 4]
```diff
@@ -17,6 +17,24 @@ class Query {
   static greaterThanEqual = (attribute, value) =>
     Query.addQuery(attribute, "greaterThanEqual", value);
 
+  static isNull = (attribute) =>
```
Comment (Meldiron) [inline lib/query.js:20]: I think we are missing type definition for this class

File: index.d.ts (hunk 3 above)
Comment (christyjacob4) [reply index.d.ts:2766]: Yea a bit concerned about this 🤔

**Comment by rossanmol** [comment]
@christyjacob4  please merge :)

**Review by rossanmol — APPROVED** [review]

File: lib/query.js (hunk 4 above)
Comment (Meldiron) [reply lib/query.js:20]: Still a valid concern. Types missing in index.d.ts

**Pull request merged**